### PR TITLE
Ukrainian translation for general strings and Main levels.

### PR DESCRIPTION
**Pull request by PanAeon**
Ukrainian translation for general strings and Main levels.
Fix css for bottom right panel to fit a new language.

### CODE REVIEW

File: src/js/react_views/IntlHelperBarView.jsx [hunk 1]
```diff
@@ -88,6 +88,12 @@ var IntlHelperBarView = React.createClass({
         this.fireCommand('locale ru_RU; levels');
       }.bind(this)
     }, {
+      text: 'Українська',
```
Comment (pcottle) [inline src/js/react_views/IntlHelperBarView.jsx]: ah good! you found this -- this is one of the two spots to update if you are adding a new locale

File: src/js/stores/LocaleStore.js [hunk 2]
```diff
@@ -20,7 +20,8 @@ var langLocaleMap = {
   fr: 'fr_FR',
   de: 'de_DE',
   pt: 'pt_BR',
-  ru: 'ru_RU'
+  ru: 'ru_RU',
+  uk: 'uk'
```
Comment (pcottle) [inline src/js/stores/LocaleStore.js]: and this is the other :)

File: src/style/main.css [hunk 3]
```diff
@@ -672,7 +672,7 @@ div.helperBar.show {
   -o-transform: translate3d(0,0,0);
   -ms-transform: translate3d(0,0,0);
   transform: translate3d(0,0,0);
-  height: 70px;
+  height: 85px;
```
Comment (pcottle) [inline src/style/main.css]: ah dang, time for bigger help bars I guess?

**Comment by pcottle** [comment]
Wow @PanAeon this is huge!! :tada: :tada: :tada: If you want you can add yourself to the list of helpful translators in the README file, but that's also been a bit out of date so I might just end up removing it.

Anyways this is is awesome, merging and pulling now

**Comment by PanAeon** [comment]
Thank you) I'm glad you like it.

**Comment by alexandear** [comment]
@PanAeon Дякую за переклад :+1: Сам хотів його зробити, але руки ніяк не доходили :)

Thanks for the Ukrainian translation. I wanted to do it but had no time :)

**Pull request merged**